.github/workflows/e2e-tests.yml

```diff
@@ -122,21 +122,28 @@ jobs:
         CONFIG_KEY: ${{ secrets.E2E_CONFIG_KEY }}
         SUITE: ${{ matrix.suite }}
         PROJECT_NAME:  ${{ matrix.project }}
+        DISPATCH_REF: ${{ github.event.client_payload.ref_name }}
       run: |
         echo "::group::Decrypt config"
         pnpm run config:decrypt
         echo "::endgroup::"
 
-        if [ "${SUITE}" != atomic ]; then
+        if [[ "${SUITE}" != atomic* ]]; then
           echo "::group::Start docker environment"
           pnpm run env:up
           echo "::endgroup::"
 
           echo "::group::Create tunnel"
           pnpm run tunnel:up
           echo "::endgroup::"
-        else
-          echo "TEST_SITE=atomic" >> $GITHUB_ENV
+        fi
+
+        if [[ "${SUITE}" == atomic* ]]; then
+          echo "TEST_SITE=${SUITE}" >> $GITHUB_ENV
+          if [[ "${SUITE}" == atomicRc ]]; then
+            # Wait for the site to pick up latest tag version (DISPATCH_REF)
+            node ./bin/wait-for-atomic-site.cjs
```

Comment:
Don't we want to run this for bleeding edge also? Maybe `wait-for-atomic-site.cjs` could receive as argument the version to force/wait for?

Reply:
Yeah, I guess it wouldn't hurt :)